spec/oas_v1.yaml

```diff
@@ -1035,10 +1426,317 @@ paths:
           content:
             application/json:
               schema: 
-                $ref: '#/components/schemas/Contract'
+                $ref: '#/components/schemas/VotingToken'
+        '400':
+          description: Bad Request
+        '401': 
+          description: Unauthorized
+        '500':
+          description: Internal Server Error
+  /rpgf/rounds/{roundId}:
+    get:
+      summary: Gets a specific RPGF round
+      description: >
+        Retrieves a specific RPGF round on Agora.
+      operationId: getRPGFRoundById
+      tags:
+        - RPGF
+        - RPGFRounds
+      parameters:
+        - name: roundId
+          in: path
+          description: The round ID of the RPGF round to retrieve.
+          required: true
+          schema:
+            type: integer
+      responses: 
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema: 
+                $ref: "#/components/schemas/RPGFRound"
+        '400':
+          description: Bad Request
+        '401': 
+          description: Unauthorized
+        '500':
+          description: Internal Server Error
+  /rpgf/rounds/{roundId}/ballots:
+  # Should this be paginated? How many ballots per round are expected?
```

Comment:
yes good call, we gonna have a couple of 100s of ballots

Reply:
btw is there a way to hide the admin only endpoints from public? Not sure if it's possible to have it in the yaml but hide it from the public docs?

Reply:
I can check if there are any OAS level primitives which might help. we could always have particular endpoints documented in a private repository. what's the concern with having these documented, if inaccessible otherwise?